extensions/jaxb/runtime/pom.xml

```diff
@@ -70,6 +70,19 @@
                     </annotationProcessorPaths>
                 </configuration>
             </plugin>
+            <!-- The entity classes needs to be indexed -->
+            <plugin>
+                <groupId>io.smallrye</groupId>
+                <artifactId>jandex-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>make-index</id>
+                        <goals>
+                            <goal>jandex</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
```

Comment:
Why is this needed? The entity classes should be in the app, not in the extension, shouldn't they?